backend/coins/btc/handlers/handlers.go

```diff
@@ -165,9 +173,9 @@ func (handlers *Handlers) getUTXOs(_ *http.Request) (interface{}, error) {
 func (handlers *Handlers) getAccountBalance(_ *http.Request) (interface{}, error) {
 	balance := handlers.account.Balance()
 	return map[string]interface{}{
-		"available":   handlers.formatAmountAsJSON(int64(balance.Available)),
-		"incoming":    handlers.formatAmountAsJSON(int64(balance.Incoming)),
-		"hasIncoming": balance.Incoming != 0,
+		"available":   handlers.formatAmountAsJSON(balance.Available),
+		"incoming":    handlers.formatAmountAsJSON(balance.Incoming),
+		"hasIncoming": balance.Incoming.Int().Cmp(big.NewInt(0)) != 0,
```

Comment:
I guess you could use the same trick here: `balance.Incoming.BigInt().Sign() > 0`.
Also, when the comparison is `-1`, is it indented that `hasIncoming` is `true`? I mean, is it possible for `Incoming` to be negative? I guess not but if for some reason there's a regression bug and it is indeed negative, it'll show `hasIncoming` as `true`.